util/actions.js

```diff
@@ -0,0 +1,27 @@
+'use server';
+import { cookies } from 'next/headers';
```

Comment:
I guess usually we leave an empty line between `'use server';` / `'use client';` and the imports below
```suggestion
'use server';

import { cookies } from 'next/headers';
```